--- PyGitUp-1.6.1/Pipfile~	2018-12-12 19:49:45.000000000 +0100
+++ PyGitUp-1.6.1/Pipfile	2019-03-24 11:25:03.081121421 +0100
@@ -7,7 +7,7 @@
 GitPython = ">=2.1.8"
 colorama = ">=0.3.7"
 termcolor = ">=1.1.0"
-click = ">=7.0.0"
+click = ">=7.0"
 six = ">=1.10.0"
 
 [dev-packages]
--- PyGitUp-1.6.1/setup.py~	2018-12-12 19:49:45.000000000 +0100
+++ PyGitUp-1.6.1/setup.py	2019-03-24 11:24:56.224366797 +0100
@@ -6,7 +6,7 @@
     version="1.6.1",
     packages=find_packages(),
     install_requires=['GitPython>=2.1.8', 'colorama>=0.3.7',
-                      'termcolor>=1.1.0', 'click>=7.0.0',
+                      'termcolor>=1.1.0', 'click>=7.0',
                       'six>=1.10.0'],
 
     # Tests
